crane/search/base.py

```diff
@@ -108,6 +108,6 @@ def _get_data(url):
 
 # this data structure should be used to return search results in a uniform
 # and well-defined way.
-class SearchResult(namedtuple('SearchResult', ['name', 'description', 'is_trusted',
+class SearchResult(namedtuple('SearchResult', ['name', 'description','should_filter', 'is_trusted',
```

Comment:
Since ``should_filter`` is a new parameter, and since it is a different type of data (it's not part of the result, but rather metadata about the result), I would make it the last parameter in this data structure.